worker/pkg/workflows/datasync/activities/benthos-builder.go

```diff
@@ -595,8 +595,18 @@ func (b *benthosBuilder) buildBenthosGenerateSourceConfigResponses(
 		if err != nil {
 			return nil, err
 		}
-		if mapping.Mutation == nil {
-			return nil, errors.New("unable to generate config mapping for table") // workshop this more
+
+		var mutationMapping string
```

Comment:
So the `buildProcessorConfig` builds both a `mutation` and `javascript` config right?

Maybe that method could be further broken down internally to look something like:

```
func buildProcessorConfig(...) {
   mutation := buildMutationConfig(...)

   js := buildJsConfig(...)
   
  // construct slice
  return processorConfigs
}
```

Then, for the context of this method, instead of calling `buildProcessorConfig` and having to iterate over the output, you could then just call:
```
mapping := b.buildMutationConfig(ctx, tableMapping.Mappings)
```

Reply:
refactored and updated